.pipelines/azure_pipeline_mergedbranches.yaml

```diff
@@ -404,9 +409,14 @@ jobs:
   - script: dir $(Build.ArtifactStagingDirectory)
     displayName: 'List files in Staging Directory'
 
-  - task: EsrpCodeSigning@3
+  - task: EsrpCodeSigning@5
     inputs:
-      ConnectedServiceName: 'AME ESRPContainerInsights Service Connection'
+      ConnectedServiceName: 'AME ESRPContainerInsights MSI Service Connection'
+      AppRegistrationClientId: 'bc3b3065-9256-4473-a288-e419f37f4d6d'
```

Comment:
@wanlonghenry , can we make these pipeline variables and pass from the pipelines instead of exposing in public GitHub code?